rocky/tools/models.py

```diff
@@ -170,6 +182,55 @@ class STATUSES(models.TextChoices):
     def blocked(self):
         return self.status == OrganizationMember.STATUSES.BLOCKED
 
+    @property
+    def is_admin(self):
+        return self.groups.filter(name=GROUP_ADMIN).exists()
+
+    @property
+    def is_redteam(self):
+        return self.groups.filter(name=GROUP_REDTEAM).exists()
+
+    @property
+    def is_client(self):
+        return self.groups.filter(name=GROUP_CLIENT).exists()
+
+    def get_permissions_codenames(self, permissions):
```

Comment:
Where are the tests for this? Based on just the code I can see that `"app_label."` also works, which contradicts with the error message